core/dbt/include/global_project/macros/materializations/models/incremental/get_incremental_predicates.sql

```diff
@@ -0,0 +1,28 @@
+{#
+
+    These macros will compile the proper join predicates for incremental models,
+    merging default behavior with the optional, user-supplied incremental_predicates
+    config. 
+
+#}
+
+{% macro get_incremental_predicates(target_relation, incremental_strategy, unique_key, user_predicates, partitions=none) %}
+    {{ adapter.dispatch('get_incremental_predicates')(target_relation, incremental_strategy, unique_key, user_predicates, partitions) }}
+{% endmacro %}
+
+{% macro default__get_incremental_predicates(target_relation, incremental_strategy, unique_key, user_predicates=none, partitions=none) %}
+    {#
+    
+        This behavior should only be observed when dbt calls the default
+        `get_delete_insert_merge_sql` strategy in dbt-core
```

Comment:
does this mean incremental predicates will only work for delete+insert strategy? is this just to get the feature started with least amount of work, or is there a blocker to implement for the merge strategy?

I don't exactly have an exact use case to share, so just curious and generally thinking about pros/cons of moving from merge to delete+insert strategy.